kanbananza-backend/src/main/controllers/schemas/user.json

```diff
@@ -1,30 +1,27 @@
 {
-  "create": {
-    "title": "new user",
-    "description": "describes properties required to create a new user and their characteristics",
-    "type": "object",
-    "properties": {
-      "firstName": {
-        "type": "string",
-        "isNotEmpty": true,
-        "description": "first name of the user"
-      },
-      "lastName": {
-        "type": "string",
-        "isNotEmpty": true,
-        "description": "last name of the user"
-      },
-      "email": {
-        "type": "string",
-        "format": "email",
-        "description": "email of the user"
-      },
-      "password": {
-        "type": "string",
-        "isNotEmpty": true,
-        "description": "password of the user"
-      }
+  "description": "all modelled properties of a user",
+  "type": "object",
+  "properties": {
+    "firstName": {
+      "type": "string",
+      "isNotEmpty": true,
+      "description": "first name of the user"
```

Comment:
These changes need be made to card.json as well to fix the create card end point.

Reply:
yep thanks!